app/files/error-handler.js

```diff
@@ -0,0 +1,13 @@
+const handleUploadError = (err, sanitizedFilename) => {
+  let simplifiedError = ''
+  if (err.message === 'Response Error: 413 Payload Too Large') {
+    simplifiedError = 'Cannot exceed 10 MB'
+  }
+  return {
+    filename: sanitizedFilename,
+    status: 'Error',
+    error: simplifiedError
```

Comment:
If the error message is not 413 this would return `''` as the simplified error, is this the intention?

Can also be refactored to use ternary operator as below which would return the original error message if its not a 413.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_operator

```suggestion
    error: err.message === 'Response Error: 413 Payload Too Large' ? 'Cannot exceed 10 MB' : err.message
```

Reply:
yeah, now I remember why I did it this way, so we can configure any other error associated with files error handling. for example if we have a bespoke error returned from cosmos, we could unpack it into simplified view. I do agree that any other error is going to be returned as '' . I can address this by adding else statement that would assing err.message to be simplifiedError. Let me know which way you want me to fix this

Reply:
Makes sense. How about this:
```suggestion
    error: simplifiedError ? simplifiedError : err.message
```
Now if simplifiedError is a string it will return the value for it otherwise if its an empty string then it will instead return the `err.message`

Reply:
Superb use ternary 👯